debian/xenial/foreman-proxy/control

```diff
@@ -10,7 +10,7 @@ Vcs-Browser: https://github.com/theforeman/smart-proxy
 
 Package: foreman-proxy
 Architecture: all
-Depends: ruby | ruby-interpreter, rake, ruby-sinatra (>= 1.3.3), ruby-rack (>= 1.1.0), ruby-json, ruby-augeas, ruby-bundler-ext, ruby-concurrent (>= 1.0.0), ruby-concurrent (<< 2.0.0)
+Depends: ruby (>= 2.3.1) | ruby-interpreter, rake, ruby-sinatra (>= 1.3.3), ruby-rack (>= 1.1.0), ruby-json, ruby-augeas, ruby-bundler-ext, ruby-concurrent (>= 1.0.0), ruby-concurrent (<< 2.0.0)
```

Comment:
Please revert this change and add a dependency onto `ruby2.3 (>= 2.3.1)` instead. This has to do with Ruby is packaged in Debian/Ubuntu and how Ubuntu is dealing with later updates.